services/submission/src/main/resources/application.yaml

```diff
@@ -39,6 +39,8 @@ services:
     maximum-request-size: ${MAXIMUM_REQUEST_SIZE:100KB}
     # The maximum rolling period accepted by the SubmissionController (e.g. 144 (24 hours))
     max-rolling-period: ${MAX_ROLLING_PERIOD:144}
+    # The minimum rolling period accepted by the SubmissionController (e.g. 1 (24 hours))
```

Comment:
I think the comment contains an error. A rolling period of 1 does not equal 24 hours but 10 minutes